src/components/onboarding/Wizard.vue

```diff
@@ -134,10 +134,8 @@ export default {
       // set step after the last to hide permanently (or until a new 'step' is added eventually):
```

Comment:
this comment is misplaced it should be for `this.setWizardCurrentStep(steps.length);` so could you please move it into the `if` too

Reply:
Sure